src/issues/issueFeatureRegistrar.ts

```diff
@@ -61,6 +66,29 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 		}
 	}
 
+	async createIssueFromFile() {
+		if (this.createIssueInfo === undefined) {
+			return;
+		}
+		let text: string;
+		if (!vscode.window.activeTextEditor) {
+			return;
+		}
+		text = vscode.window.activeTextEditor.document.getText();
+		const indexOfEmptyLine = text.indexOf('\n\n');

```

Comment:
is it at all possible that the line ending would be changed on save or anything like that?

Reply:
Yes, added some handling for that.